krun.py

```diff
@@ -220,6 +223,10 @@ def inner_main(mailer, on_first_invocation, config, args):
         util.fatal("Instrumentation dir '%s' exists (from an older run?)" %
                    (instr_dir))
 
+    if envlog_dir_exists and on_first_invocation:
+        util.fatal("Env log dir '%s' exists (from an older run?)" %
```

Comment:
I'm going to stick with just "Env log dir '%s' already exists". If this fires, then krun's internal consistency is broken and the user is on their own to decide what to do.